ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterManager.java

```diff
@@ -770,24 +850,28 @@ private void ensurePartitionAbsenceOnNodeAndValidateCapacity(AmbryPartition part
   /**
    * Class that stores all the information associated with a datacenter.
    */
-  private static class DcInfo {
+  static class DcInfo {
     final String dcName;
     final DcZkInfo dcZkInfo;
     final HelixManager helixManager;
     final ClusterChangeHandler clusterChangeHandler;
+    final RoutingTableProvider routingTableProvider;
```

Comment:
where is this used?

Reply:
Will remove. Thanks for reminding me.

Reply:
Will remove. Currently no methods require such info.  Thanks for pointing it out.